utility/firebase.js

```diff
@@ -746,3 +746,42 @@ export const updateApplicantTags = async (userId, applicantTags) => {
     .doc(userId)
     .update({ applicantTags })
 }
+
+// hacker application questions specific
+export const getHackerAppQuestions = async (selectedHackathon, category) => {
+  const data = await db.collection('HackerAppQuestions').doc(selectedHackathon.slice(0, -4)).collection(category).get()
+  return data.docs.map(doc => doc.data())
+}
+
+export const updateHackerAppQuestions = async (selectedHackathon, questions, category) => {
+  const hackathonRef = db.collection('HackerAppQuestions').doc(selectedHackathon.slice(0, -4))
+  const categoryRef = hackathonRef.collection(category)
+
+  const batch = db.batch()
+
+  // clear all
+  const existingDocs = await categoryRef.get()
+  existingDocs.forEach(doc => {
+    batch.delete(doc.ref)
+  })
+
+  questions.forEach((question, index) => {
+    const newDocRef = categoryRef.doc(`${index}`)
```

Comment:
probably not a big issue, but iirc firestore sorts IDs in a collection lexicographically ("1", "2", "10", "11" would be sorted as "1", "10", "11", "2"), so adding a padding to the index ("001") would help mitigate that

Reply:
jk cmd-f had 20 questions in basic info this year LOL